src/main/java/it/gov/pagopa/payhub/activities/service/treasury/TreasuryBaseOpiHandlerService.java

```diff
@@ -0,0 +1,29 @@
+package it.gov.pagopa.payhub.activities.service.treasury;
+
+import it.gov.pagopa.payhub.activities.xsd.treasury.opi14.FlussoGiornaleDiCassa;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class TreasuryBaseOpiHandlerService {
```

Comment:
Don't do this tò retrieve dynamically classes! Spring allow to inject beans using Generic thus this class could be removed (moreover you were using @SuppressWarning which should be avoided! Using It could be and hint of bad design)